### ISSUE TITLE
Let's Encrypt HA Rancher is broken in master-head

### ISSUE THREAD

**Issue by aaronRancher** [rancher/rancher#25302]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
Install HA Rancher with Let's Encrypt (https://rancher.com/docs/rancher/v2.x/en/installation/k8s-install/create-nodes-lb/):
Provision instances
Create config file
rke up
Install cert manager
Install Rancher with Let's Encrypt:
```
helm install rancher rancher-alpha/rancher \
  --namespace cattle-system \
  --set hostname=<domain name> \
  --set ingress.tls.source=letsEncrypt \
  --set letsEncrypt.email=<email> --version 2.4.0-alpha1 --set rancherImageTag=master-head
```

**Result:**
Rancher is installed successfully, and the setup can be accessed. But the cert is invalid.
![Screen Shot 2020-02-06 at 4 07 06 PM](https://user-images.githubusercontent.com/57409052/73986520-08367800-48fb-11ea-989d-e6b582528a98.png)
Cluster creation also fails with: `Cluster must have at least one etcd plane host: failed to connect to the following etcd host(s) []`

**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
`master-head` `1a03ecea3`
- Installation option (single install/HA): HA

**Comment by aaronRancher** [comment]
Related Let's Encrypt issue: #25301

**Comment by aaronRancher** [comment]
The bug fix,https://github.com/rancher/rancher/commit/19c946de352c77e1ab04184b8345ecd6021e33b4 , is verified with Rancher `master-head` version `1a03ecea3`.

Clusters are provisioned successfully, logs and exec shells are functional.

**Comment by maggieliu** [comment]
Here is the PR for this issue: https://github.com/rancher/rancher/pull/24509

**Comment by rawmind0** [comment]
Creating a new alpha tag should address the issue, due to patch is already merged on master

**Comment by deniseschannon** [comment]
Available to test with v2.4.0-rc1

**Comment by aaronRancher** [comment]
The bug fix is verified with Rancher server `rancher\rancher: v2.4.0-rc1`

Steps:
Install HA Rancher with Let's Encrypt (https://rancher.com/docs/rancher/v2.x/en/installation/k8s-install/create-nodes-lb/):
Provision instances
Create config file
rke up
Install cert manager
Install Rancher with Let's Encrypt:
```
helm install rancher rancher-latest/rancher \
  --namespace cattle-system \
  --set hostname=<hosname> \
  --set ingress.tls.source=letsEncrypt \
  --set letsEncrypt.email=<email>--version v2.4.0-rc1
```

Results:
Certificates are valid
![Screen Shot 2020-02-27 at 1 05 15 PM](https://user-images.githubusercontent.com/57409052/75483275-c03bcb80-5963-11ea-9db1-20695889f420.png)
Cluster provisioning succeeds with providers and custom clusters.
